test/data/images/images.tsv

```diff
@@ -1,3 +1,4 @@
+tomato.bmp	tomato
```

Comment:
>. [](start = 6, length = 1)

did you intend to add a test, or the test using this file is already defined?

Reply:
There are existing tests that use this file which cover the new functionality after adding this image.

---
In reply to: [238835568](https://github.com/dotnet/machinelearning/pull/1821#discussion_r238835568) [](ancestors = 238835568)